deno/deepgram-transcribe-video/src/mod.ts

```diff
@@ -0,0 +1,53 @@
+export default async function (req: any, res: any) {
+  const APIkey = req.variables["DEEPGRAM_API_KEY"];
+  const fileUrl = JSON.parse(req.payload)["fileUrl"];
+
+  // error handling
+  if (!APIkey) {
+    res.json({
+      success: false,
+      message: "Missing Deepgram API key",
+    });
+  }
```

Comment:
you need to make sure to call return or else the rest of the code will still execute.